templates/repo/settings/webhook/settings.tmpl

```diff
@@ -248,6 +248,13 @@
 	<span class="help">{{.locale.Tr "repo.settings.branch_filter_desc" | Str2html}}</span>
 </div>
 
+<!-- Authorization Header -->
+<div class="field{{if eq .HookType "matrix"}} required{{end}}">
+	<label for="authorization_header">{{.locale.Tr "repo.settings.authorization_header"}}</label>
+	<input id="authorization_header" name="authorization_header" type="text" value="{{ .Webhook.HeaderAuthorization }}"{{if eq .HookType "matrix"}} placeholder="Bearer $access_token" required{{end}}>
+	<span class="help">{{.locale.Tr "repo.settings.authorization_header_desc" "<code>Bearer token123456</code>, <code>Basic YWxhZGRpbjpvcGVuc2VzYW1l</code>" | Str2html}}</span>
```

Comment:
I think help icon with tooltip is generally preferred because it does not litter the view. But exceptions can be made for example for stuff that is useful to copy.